### PR TITLE
Fix cob41 bringup

### PR DESCRIPTION
**Pull request by fmessmer**
head pc currently not used - as long as Windows NUC is mounted

### CODE REVIEW

File: cob_bringup/robots/cob4-3.xml [hunk 1]
```diff
@@ -65,12 +67,6 @@
 		<include file="$(find cob_bringup)/tools/robot_state_publisher.launch">
 			<arg name="robot" value="$(arg robot)" />
 		</include>
-
-		<!-- scan unifier -->
-		<include file="$(find cob_bringup)/drivers/scan_unifier.launch" >
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-3.xml:70]: why do you delete the scan_unifier?

Comment (NadiaHG) [inline cob_bringup/robots/cob4-3.xml:70]: oh, my bad, just moved..

File: cob_bringup/robots/cob4-2.xml [hunk 2]
```diff
@@ -231,7 +229,6 @@
 	<group>
 		<machine name="cob4-2-h1" address="$(arg cob4-2-h1)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-2.xml:230]: should not be also commented? cob4-2 is the default setup of the cob4-1/cob4-2 robot, at least upstart and the command  cob-start are launching cob4-2 , if a pc machine can not be found the launch will die intermediately

File: cob_bringup/robots/cob4-3.xml (hunk 1 above)
Comment (fmessmer) [inline cob_bringup/robots/cob4-3.xml:70]: yes I moved it together with the scanners...as that was more logical to me...

File: cob_bringup/robots/cob4-2.xml (hunk 2 above)
Comment (floweisshardt) [inline cob_bringup/robots/cob4-2.xml:230]: +1 for applying cob4-1 changes to cob4-2, too.

Comment (fmessmer) [inline cob_bringup/robots/cob4-2.xml:230]: done

**Comment by NadiaHG** [comment]
FR: OK

**Pull request merged**